### ISSUE TITLE
Qt Creator 4 OSX add -l flag without argument

### ISSUE THREAD

**Issue by techtim** [openframeworks/openFrameworks#5139]
On 9.3 tag and on master I get the same
clang: error: argument to '-l' is missing (expected 1 value)

**Comment by Daandelange** [comment]
The [Qt Creator setup guide](http://openframeworks.cc/setup/qtcreator/) states:
> The current openFrameworks release (0.9.3) requires qtcreator 3.6.1. The nightly builds support version 4.1.
After testing, this error happens in both Qt Creator `4.0.2` and `4.1` in the linker stage but I can't locate when that `-l` flag is added to the end of the linker command. (Any idea @arturoc ?)
It happens in both projects generated with the PG and with the Qt setup modal, and both current branches `0.9.4` and `master`.

Below is part of the linker command in Qt 3.6.1 which seems to take `-framework` as the `-l` argument resulting in `Accelerate` to be interpreted as a directory to include...
````
[...] /openFrameworks/apps/myApps/emptyQtExampleBis/../../../libs/poco/lib/osx/PocoFoundation.a -l -framework Accelerate -framework AGL [...]
````

**Comment by fx-lange** [comment]
with clang and qtcreator 3.6.1 I have the same issue as @Daandelange described. this behaviour results in 

`clang: error: no such file or directory: 'Accelerate'`

for 094 and current master. stable branch works though

**Comment by myildirim** [comment]
With Qt Creator 3.6.1 I have the same problem.

**Comment by Daandelange** [comment]
@arturoc fixed this in 0b2082c18026a704c05304ba2f3c9775ee22ff2d 👍 
Works on OS X using Qt 4.1.0 _(and I think 0.9.4 is not compatible with Qt 3.6.1 anymore )_

**Comment by fx-lange** [comment]
awesome - works perfectly for me (master with qtcreator 4.1). thanks @arturoc & @Daandelange
